src/components/content/catalog/services/details/ServiceProvider.tsx

```diff
@@ -151,6 +151,8 @@ function ServiceProvider({
 
     const reRegisterRequest = useReRegisterRequest(activeServiceDetail?.serviceTemplateId ?? '');
 
+    const serviceTemplateDetailsVo = useGetServiceTemplateDetailsById(reRegisterRequest.data?.serviceTemplateId ?? '');
```

Comment:
can we avoid these changes in this PR?  Wang is working on these changes in his PR.. you can also check with him once.

Reply:
like we talked, keep this and make new PR rebase this.